exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,3564 @@
+{
```

Comment:
<p>typo: ".. the column"</p><p> </p><p>"This enables you to trace back Documents(and their associated results) to the exact column and of our inpu data view/table the originated from" Please re-write this sentence.</p><p> </p><p>a "step" table, and a "lookup" table and a "setup" table. What is "step" table. Can you use one "and" please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>There is also a "DOCUMENTS_AI_LAB_MY_VIEW" table, which contains IDs of the input text and documents, as well as the <strong>name of the </strong>column the input text originated from.</p><p> </p><p>"a "&lt;&lt;step-name&gt;&gt;" table, a "lookup" table and a "setup" table " this is still puzzling. I would say "A preprocessing step stores its output in one or more tables. Multiple tables are required because the output content is usually normalised.". For the record, the support table names a formatted like &lt;main_table_name&gt;_LOOKUP_&lt;normalized_column_name&gt;.</p>